src/pyhf/readxml.py

```diff
@@ -13,16 +25,32 @@
 from pyhf import compat
 from pyhf import exceptions
 from pyhf import schema
+from pyhf.typing import (
+    Sample,
+    Parameter,
+    Modifier,
+    LumiSys,
+    NormSys,
+    NormFactor,
+    HistoSys,
+    StatError,
+    ShapeSys,
+    ShapeFactor,
+    Measurement,
+    ParameterBase,
+    Channel,
+    Observation,
+    Workspace,
+    PathOrStr,
+)
```

Comment:
This is just a suggestion to avoid more import movement noise if we end up eventually turning on `isort` in `pre-commit`. Feel free to ignore though.

```suggestion
from pyhf.typing import (
    Channel,
    HistoSys,
    LumiSys,
    Measurement,
    Modifier,
    NormFactor,
    NormSys,
    Observation,
    Parameter,
    ParameterBase,
    PathOrStr,
    Sample,
    ShapeFactor,
    ShapeSys,
    StatError,
    Workspace,
)
```